reggiedata/patches/NewerSMBW/spritedata.xml

```diff
@@ -710,32 +437,10 @@
-    <list bit="23-24" title="Delay on Multi-Use" comment="This determines how long the switch will take to pop up after the timer expires on a Timed Multi-Use switch." requiredbit="28, 40" requiredval="0, 1">
-      <entry value="0">Instant</entry>
-      <entry value="2">Short Delay</entry>
-      <entry value="3">Long Delay</entry>
-      <entry value="1">Very Long Delay</entry>
-    </list> <!-- Appears only if Timed Event AND Multi-Use are set -->
-    <checkbox bit="18" title="Loads at Midway" comment="When enabled, the switch will only appear if the level has been loaded from a Midway point."/>
-    <value bit="1-8" title="Triggering Event ID" comment="This switch will not be visible if a Triggering Event is defined unless that event is activated." requiredbit="40" requiredval="0" idtype="Triggering Event"/> <!-- Disappears if Multi-Use is set -->
-    <value bit="1-8" title="Triggering Event ID" comment="This switch will not be visible if a Triggering Event is defined unless that event is activated." advanced="True" advancedcomment="Using a Triggering ID with Multi-Use functionality is somewhat glitchy and not recommended." requiredbit="40" idtype="Triggering Event"/> <!-- Appears if Multi-Use is set, but only if Advanced fields are enabled -->
-  </sprite> <!-- #41: "P" Switch (Blue) -->
   
   <sprite id="42" name="&quot;!&quot; Switch (Red)" asmhacks="True" sizehacks="True" notes="This switch is similar to the ? Switch, but it has the additional effect of changing all outlined Red Blocks (Pa1_ Object 57) into solid Red Blocks. This also applies to any outlined Red Blocks spawned by a Tile God. This additional functionality only occurs when using the Timed Event mode, but a Target Event is not required to make use of this functionality.">
     <dependency notes="Activates 'Red Block Outline A/B' tiles."/>
-    <list bit="29-32" title="Color" comment="This determines what color the switch is. Different colored switches do not activate different colors of blocks." comment2="[Bug] All color values other than the original will spawn one block further upwards than it should.">
+    <list bit="29-32" title="Color" comment="This determines what color the switch is. Different colored switches do not activate different colors of blocks." comment2="[Bug] All color values other than the original will spawn one block further upwards than it should. this is accounted for in the sprite image.">
```

Comment:
[...] it should. this is accounted for in the sprite image.">
`this` should be capitalised.